### PR TITLE
Update tip to include Bech32 address format

### PR DESCRIPTION
**Pull request by tochicool**

### CODE REVIEW

File: ch01.asciidoc [hunk 1]
```diff
@@ -133,7 +133,7 @@ image::images/mbc2_0101.png["MyceliumWelcome"]
```
Comment (privsenotes) [inline ch01.asciidoc:133]: X

File: ch01.asciidoc [hunk 2]
```diff
@@ -133,7 +133,7 @@ image::images/mbc2_0101.png["MyceliumWelcome"]
 
 [TIP]
```
Comment (privsenotes) [inline ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X stop

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

File: ch01.asciidoc (hunk 1 above)
Comment (privsenotes) [reply ch01.asciidoc:133]: X

Comment (privsenotes) [reply ch01.asciidoc:133]: X

Comment (privsenotes) [reply ch01.asciidoc:133]: X

File: ch01.asciidoc (hunk 2 above)
Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

Comment (privsenotes) [reply ch01.asciidoc:134]: X

**Review by wbnns — APPROVED** [review]
@Tochicool 

Thank you.

**Pull request merged**